frontend/src/components/GTimeTextField.vue

```diff
@@ -0,0 +1,91 @@
+<!--
+SPDX-FileCopyrightText: 2023 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+-->
+
+<template>
+  <v-text-field
+    v-model="timeValue"
+    :error-messages="errorMessages"
+    :hint="inputHint"
+    :type="textFieldType"
+    @change="v$.timeValue.$touch()"
+  />
+</template>
+
+<script setup>
+
+import { computed } from 'vue'
+import { useVuelidate } from '@vuelidate/core'
+import { useVModel } from '@vueuse/core'
+
+import { getValidationErrors } from '@/utils'
+
+const props = defineProps({
+  modelValue: {
+    type: String,
+  },
+  errorMessages: {
+    type: Array,
+  },
+  hint: {
+    type: String,
+  },
+})
+
+const emit = defineEmits([
+  'update:modelValue',
+])
+
+const timeValue = useVModel(props, 'modelValue', emit)
+
+const isSafari = computed(() => {
```

Comment:
Try to detect the specific behavior you're trying to fix, instead of targeting it with `isSafari`?

Reply:
It seems not possible to check for the behavior!